pkg/utils/syscalls/arm64/syscalls.go

```diff
@@ -0,0 +1,918 @@
+	"sigsuspend":                   -1,
+	"socket":                       198,
+	"socketcall":                   -1,
+	"socketpair":                   199,
+	"splice":                       76,
+	"spu_create":                   -1,
+	"spu_run":                      -1,
+	"ssetmask":                     -1,
+	"stat":                         -1,
+	"stat64":                       -1,
+	"statfs":                       43,
+	"statfs64":                     -1,
+	"statx":                        291,
+	"stime":                        -1,
+	"subpage_prot":                 -1,
+	"swapcontext":                  -1,
+	"swapoff":                      225,
+	"swapon":                       224,
+	"switch_endian":                -1,
+	"symlink":                      -1,
+	"symlinkat":                    36,
+	"sync":                         81,
+	"sync_file_range":              84,
+	"sync_file_range2":             -1,
+	"syncfs":                       267,
+	"sys_debug_setcontext":         -1,
+	"syscall":                      -1,
+	"sysfs":                        -1,
+	"sysinfo":                      179,
+	"syslog":                       116,
+	"sysmips":                      -1,
+	"tee":                          77,
+	"tgkill":                       131,
+	"time":                         -1,
+	"timer_create":                 107,
+	"timer_delete":                 111,
+	"timer_getoverrun":             109,
+	"timer_gettime":                108,
+	"timer_gettime64":              -1,
+	"timer_settime":                110,
+	"timer_settime64":              -1,
+	"timerfd":                      -1,
+	"timerfd_create":               85,
+	"timerfd_gettime":              87,
+	"timerfd_gettime64":            -1,
+	"timerfd_settime":              86,
+	"timerfd_settime64":            -1,
+	"times":                        153,
+	"tkill":                        130,
+	"truncate":                     45,
+	"truncate64":                   -1,
+	"ugetrlimit":                   -1,
+	"umask":                        166,
+	"umount":                       -1,
+	"umount2":                      39,
+	"uname":                        160,
+	"unlink":                       -1,
+	"unlinkat":                     35,
+	"unshare":                      97,
+	"uselib":                       -1,
+	"userfaultfd":                  282,
+	"ustat":                        -1,
+	"utime":                        -1,
+	"utimensat":                    88,
+	"utimensat_time64":             -1,
+	"utimes":                       -1,
+	"utrap_install":                -1,
+	"vfork":                        -1,
+	"vhangup":                      58,
+	"vm86":                         -1,
+	"vm86old":                      -1,
+	"vmsplice":                     75,
+	"wait4":                        260,
+	"waitid":                       95,
+	"waitpid":                      -1,
+	"write":                        64,
+	"writev":                       66,
+}
+
+var SyscallsNumberToName = map[int]string{
```

Comment:
nit: is it possible to get this ordered by number?

Reply:
Any special reason for this?

Reply:
I was thinking that it'll be easier to understand what is the latest syscall that was added, but `syscallsNameToNumber` is ordered by name so that's good enough.